feat(java-lombok): Add documentation how to enable Lombok in Java workspaces (#1912)
codenvy/che-docs: modules/extensions/partials/proc_java-lombok.adoc

[id="proc_java-lombok_{context}"]
= Java Lombok

[role="_abstract"]
This section shows how to enable the Lombok support in your Java projects.

> Project Lombok is a Java library that automatically plugs into your editor and build tools, spicing up your Java.

Reviewer: ```suggestion
Project Lombok is a Java library that automatically plugs into your editor and build tools, spicing up your Java environment.
```

> Never write another getter or equals method again, with one annotation your class has a fully featured builder, Automate your logging variables, and much more.

Reviewer: Not sure if we need this line, it might be better to just provide a link to the Lombok homepage -- let them sell the project.

Author: OK I will remove these lines.

> By default, `lombok.jar` is available in all the Java plugins provided by {prod-short}

Reviewer: ```suggestion
By default `lombok.jar` is available in all Java plugins provided by {prod-short}
```

> To enable Lombok in a {prod-short} workspace, use the instructions in this section.

Reviewer: ```suggestion
To enable Lombok in a {prod-short} workspace, see the instructions below.
```

.Prerequisites
A workspace or a devfile with

* One of the Java based plugin enabled (`redhat/java`, `redhat/java11`, `redhat/java8`, `redhat/quarkus-java8` or `redhat/quarkus-java11`),
* A valid Lombok project to import.

.Procedure

. Open the workspace devfile

> . Edit the existing Java plugin, adding the preferences:

Reviewer: ```suggestion
. Edit the existing Java plugin section, adding the preference:
```
